Obedient-Lobsters/WorkforceManagement
# Create Employee: Controller, Model, View

# Description
The user can now navigate to Create Employee form by clicking the "Create New" link on the View All Employee View. The form allows the user to add an employee first name, last name, email, start date, department, and supervisor status. It is posted to the database when the user clicks "Create" button. The user is redirected to view all employee page once the form is completed. 
 Fixes #9 

 ## Type of change

 # Testing Instructions
If you need to create the database:

Start SQL Server Management Studio

Run BangazonDatabaseCreateSeed.sql in SSMS

Update the appsetting.json in Visual Studio community to reflect your desktop and your server

If your database is already seeded or has completed the steps above:

In the terminal, `git fetch --all` and `git checkout lg-AddEmployee` to switch branches

Start the program by cd'ing into the WorkforceManagement (make sure you are deep enough, the .sln file is a few WorkforceManagement directories deep) and using the command `start WorkforceManagement.sln`.

Press F5 to start the program in Visual Studio

Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the Employees link in the navbar, and you should see a list with all Employees in the database.

Click on the "Create New" link. Fill the form out with the new employee info and click "Create". A new employee is created in the database and you should be rerouted back to view all list with the new employee listed.

 # Checklist:
